### PR TITLE
[GPTQ] Add GPTQ python implement.

### PR DESCRIPTION
**Pull request by changqi1**

### CODE REVIEW

**Comment by Duyi-Wang** [comment]
Please add license head in new files.

**Comment by changqi1** [comment]
> Please add license head in new files.

Added.

**Comment by changqi1** [comment]
@pujiang Add only Python code. Will request GPTQ cpp code in another PR.

File: tools/gptq/run_weight_quant.py
Comment (pujiang2018) [inline tools/gptq/run_weight_quant.py:1]: Do we really need this file?

Comment (changqi1) [reply tools/gptq/run_weight_quant.py:1]: Let me move it into readme file.

Comment (changqi1) [reply tools/gptq/run_weight_quant.py:1]: Done.

**Review by pujiang2018 — APPROVED** [review]

**Pull request merged**